src/user/organization/popups/EditOrg.js

```diff
@@ -0,0 +1,138 @@
+import React, {Component} from 'react';
+import { withStyles } from '@material-ui/core/styles';
+import {Dialog, DialogContent, DialogTitle, Button, Grid} from '@material-ui/core';
+import {TextField, InputLabel} from '@material-ui/core';
+import IconButton from '@material-ui/core/IconButton';
+import CloseIcon from '@material-ui/icons/Close';
+import "../../profile/popups/popups.scss";
+
+export class EditOrg extends Component{
+    constructor(props){
+        super(props);
+    }
+    render(){
+      const styles = theme => ({
+        root: {
+          margin: 0,
+          padding: theme.spacing(2),
+        },
+        closeButton: {
+          position: 'absolute',
+          right: theme.spacing(1),
+          top: theme.spacing(1),
+          color: theme.palette.grey[500],
+        },
+      });
+      
+     const IconClose = withStyles(styles)(props => {
+        const { children, classes, onClose, ...other } = props;
+        return (
+          <div>
+            
+              <IconButton aria-label="close" className={classes.closeButton} onClick={this.props.onClose}>
+                <CloseIcon />
+              </IconButton>
+            
+          </div>
+        );
+      });
+      
+    return (
+      <div>
+        <Dialog
+        {...this.props}
+        size="sm"
+        aria-labelledby="customized-dialog-title">
+          <DialogTitle> 
+            <span className="title">New Organization</span>
```

Comment:
@theprover97 if its your pop-up for edit organization then why is it saying new Organization?